examples/crowdfunding/scripts/common/common.js

```diff
@@ -0,0 +1,13 @@
+async function tryExecuteTx(deployer, txnParams) {
```

Comment:
why do we have another tryExecuteTx? Could we make the use of the other function? maybe we can move it to utils and reuse it everywhere?

Reply:
Yeah I thought the same, but we should keep the examples independent of each other since if a use copies an example folder to use in their own code, they should easily do it rather than also copying an outside folder (which they will release after the code throws an error that it is missing)